Assets/Scripts/Editor/MSP CI/ProjectBuilder.cs

```diff
@@ -51,20 +79,23 @@ public static void MyBuild()
             BuildPipeline.BuildPlayer(EditorBuildSettings.scenes, path + "/Windows_Dev/msp_dev.exe", EditorUserBuildSettings.activeBuildTarget, BuildOptions.Development);
         }
     }
+	
+	private static string GetArg(string name)
+	{
+		var args = System.Environment.GetCommandLineArgs();
+		for (int i = 0; i < args.Length; i++)
+		{
+			if (args[i] == name && args.Length > i + 1)
+			{
+				return args[i + 1];
```

Comment:
do you check if the next parameter starts with `-` ?
what if I put `program.exe -customBuildPath -anotherParam

Reply:
The instruction is never going to be manually written by a user but it's hardcoded by us in the Jenkinsfile.
This means that it's not prone to errors except in case of a modification of the instruction in the Jenkinsfile, that will be promptly caught by the automatic execution of the build pipeline after pushing the changes to the PR the developer created.